sample/Determinant.js

```diff
@@ -0,0 +1,29 @@
+
+if( typeof module !== 'undefined' )
+    require( 'wmathmatrix' );
+
+var _ = wTools;
+
+var matrix3x3 = _.Matrix.make([ 3,3 ]).copy
+([
+    +1, +2, +3,
+    +0, +4, +5,
+    +0, +0, +6,
+]);
+
+var determinant = matrix3x3.determinant();
+var expected = 24;
+console.log( 'determinant of matrix3x3\n' + determinant);
+console.log( 'expected\n' + expected );
```

Comment:
```suggestion
console.log( 'determinant of matrix3x3\n' + determinant);// log "24"
```
Please add output which the user will see as the result of execution.